plugins/inputs/sqlserver/azuresqlqueries.go

```diff
@@ -437,8 +437,17 @@ WITH PerfCounters AS (
 				THEN d.[name]
 			ELSE d.[physical_database_name]
 		END
-	WHERE
-		counter_name IN (
+	WHERE 
+		spi.instance_name NOT IN (
+			'master'
+			,'model'
+			,'model_masterdb'
+			,'model_userdb'
+			,'msdb'
+			,'mssqlsystemresource'
+			,'tempdb'
```

Comment:
We want Tempdb and master for sure, others are good to exclude as this is Single DB offering so doesn't make sense. However Tempdb is critical and there are background tasks counted in master.

Reply:
Sure, added exclusions for other DBs except master and tempdb. Thanks!